src/lib/server/InputFilter.h

```diff
@@ -166,7 +166,22 @@ class InputFilter {
         String                    m_screen;
         IEventQueue*            m_events;
     };
-    
+
+    // ToggleScreenAction
+    class ToggleScreenAction : public Action {
+    public:
+        ToggleScreenAction(IEventQueue* events);
+
+        // Action overrides
+        virtual Action*        clone() const;
+        virtual String            format() const;
+        virtual void            perform(const Event&);
```

Comment:
Inconsistent indentation. Could we start the identifier immediately after the type name? This will avoid the need to reindent whenever a long type name is used.

Reply:
This is also to match the code above and below, to change that, need to change lots of lines, should be in another commit.

Reply:
I don't ask you to change everything, just the new code you're adding. It needs reformatting anyway as it's inconsistent even to itself when viewed through GitHub UI.